### PR TITLE
Create south-african-journal-of-animal-science-csl

### PR DESCRIPTION
**Pull request by Yusufras**

### CODE REVIEW

**Comment by rmzelle** [comment]
Let us know if you need help.

**Comment by Yusufras** [comment]
What is it that i am doing wrong?

**Comment by rmzelle** [comment]
See http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2FYusufras%2Fstyles%2F627babf0f41ea4b042e3449617d12e3a92966f7a%2FSouth-African-Journal-of-Animal-Science.csl&version=1.0.1 for the schema validation errors.

To validate,

```
    <contributor>
      <name>Johan Asplund</name>
      <email>asplundj@gmail.com</email>
      <name>Yusuf Ras</name>
      <email>yr@sun.ac.za</email>
    </contributor>
```

should be 

```
    <contributor>
      <name>Johan Asplund</name>
      <email>asplundj@gmail.com</email>
    </contributor>
    <contributor>
      <name>Yusuf Ras</name>
      <email>yr@sun.ac.za</email>
    </contributor>
```

while

```
<name delimiter=", " &="text"
```

should be

```
<name delimiter=", " and="text"
```

(there might be additional errors, but the validator trips over the last one)

**Comment by rmzelle** [comment]
The style also needs to follow all our style requirements (https://github.com/citation-style-language/styles/wiki/Style-Requirements): file name should be "south-african-journal-of-animal-science.csl" (lowercase), style ID and "self" link should use "http://www.zotero.org/styles/south-african-journal-of-animal-science", and the "template", "documentation" links and ISSNs needs to be updated.

(and with an updated "template" link, we prefer it if you remove any contributors or authors from the original style)

File: south-african-journal-of-animal-science.csl [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
-    <title>South African Journal of Animal Science</title>
+    <title>south-african-journal-of-animal-science.csl</title>
```
Comment (rmzelle) [inline south-african-journal-of-animal-science.csl]: The title should have remained `<title>South African Journal of Animal Science</title>`.

File: south-african-journal-of-animal-science.csl [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     <title-short>South African Journal of Animal Science</title-short>
-    <id>http://csl.mendeley.com/styles/21406231/South-African-Journal-of-Animal-Science</id>
-    <link href="http://www.zotero.org/styles/poultry-science" rel="self"/>
-    <link href="http://www.zotero.org/styles/asa-cssa-sssa" rel="template"/>
```
Comment (rmzelle) [inline south-african-journal-of-animal-science.csl]: We would still like to have a "template" link. It looks like that should be `<link href="http://www.zotero.org/styles/poultry-science" rel="template"/>`, if you indeed based this style off of poultry-science.csl.

File: south-african-journal-of-animal-science.csl [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" &="text" delimiter=", " initialize-with=""/>
```
Comment (rmzelle) [inline south-african-journal-of-animal-science.csl]: The `&="text"` here should also be `and="text"`.

**Comment by Yusufras** [comment]
What is the one issue that failed?

**Comment by rmzelle** [comment]
See http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcitation-style-language%2Fstyles%2F6abdad91c92b287804df85a464c6fec81c09a474%2Fsouth-african-journal-of-animal-science.csl&version=1.0.1

You've used `<et-al/>` as a direct child element of `<citation/>`, which is not allowed:

```
  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year-suffix">
    <et-al font-style="italic"/>
```

If you wish to italicize "et al.", you need to place `<et-al/>` below `<names/>`. See http://docs.citationstyles.org/en/stable/specification.html#et-al

**Comment by rmzelle** [comment]
(now that the style validates, our test system started noticing that the ISSN and eISSN are still for the Poultry Science journal)

**Comment by rmzelle** [comment]
Try accepting my pull request at https://github.com/Yusufras/styles/pull/2. Should fix the remaining issues.

**Comment by rmzelle** [comment]
Thanks! Sorry this took a while!

**Comment by Yusufras** [comment]
Thanks for all the help, much appreciated.

**Pull request merged**